site/en/r2/tutorials/generative/dcgan.ipynb

```diff
@@ -1,143 +1,197 @@
 {
```

Comment:
<p>Typo: "updat" --&gt; "update" </p><p><br></p><p>Maybe: "So we don't update the batchnorm parameters" --&gt; "So all layers run in inference mode (batchnorm)."</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
<p>Done</p>